include/travis/mock_build_test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -xe
+
+# allow caller to override MAPPED_DIR, but default if they don't
+MAPPED_DIR="${MAPPED_DIR:-/build}"
+
+echo 'travis_fold:start:yum'
+yum -y install git mock rpm-build ed sudo make rpmdevtools python-setuptools rpmlint
+echo 'travis_fold:end:yum'
+
+# add our repos to the mock configuration
+ed <<"EOF" /etc/mock/default.cfg
+$i
+
+[copr-be.cloud.fedoraproject.org_results_managerforlustre_manager-for-lustre_epel-7-x86_64_]
+name=added from: https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-for-lustre/epel-7-x86_64/
+baseurl=https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-for-lustre/epel-7-x86_64/
+enabled=1
+.
+wq
+EOF
+
+eval export "$(grep -e "^TRAVIS=" -e "^TRAVIS_PULL_REQUEST_BRANCH=" "$MAPPED_DIR"/travis_env)"
+
+groupadd --gid "$(stat -c '%g' "$MAPPED_DIR")" mocker
+useradd --uid "$(stat -c '%u' "$MAPPED_DIR")" --gid "$(stat -c '%g' "$MAPPED_DIR")" mocker
+usermod -a -G mock mocker
+
+if ! su - mocker <<EOF; then
+set -xe
+cd "$MAPPED_DIR"
+make rpmlint
```

Comment:
It would be nicer if this were a top-level test in it's own stage, rather than being overloaded into another test.  I wonder if it's worth an effort for us to [back]port some stuff to Trusty (thinking _rpmlint_ and _mock_ for starters) so that we can do morem natively, in Travis' Trusty Linux environment.

Do we have any idea how long Travis will continue with Trusty, or at least not provide an option to use Xenial, where more of these tools are available?

Reply:
> Do we have any idea how long Travis will continue with Trusty, or at least not provide an option to use Xenial, where more of these tools are available?

I looked the other day, they said Xenial was right around the corner here: https://blog.travis-ci.com/2017-12-01-new-update-schedule-for-linux-build-images